canonical/snapcraft.io
# Add exchange dashboard macaroon and remove candid

## Done
- Remove candid login
- Remove `candid-login-required decorator`
- Add `developer-token` gotten from exchanging macaroon to login session
## How to QA
- Go to https://snapcraft-io-4456.demos.haus/login
- Then try accessing any model service route (E.g https://snapcraft-io-4456.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models), ensure that it does not redirect and you are able to access the routes
## Issue / Card https://warthogs.atlassian.net/browse/WD-7211
Fixes #

## Screenshots
